app/dashboard/views.py

```diff
@@ -1130,6 +1131,56 @@ def social_contribution_modal(request):
     return TemplateResponse(request, 'social_contribution_modal.html', params)
 
 
+@csrf_exempt
+@require_POST
+def invite_to_bounty_based_on_skills(request):
+    """Invite users with matching skills to a bounty.
+
+    Args:
+        bounty_id (int): The primary key of the bounty to be accepted.
+        skills (string): Comma separated list of matching keywords.
+
+    Raises:
+        Http403: The exception is raised if the user is not authenticated or
+                 the args are missing.
+        Http401: The exception is raised if the user is not a staff member.
+
+    Returns:
+        Http200: Json response with {'status': 200, 'msg': 'email_sent'}.
+
+    """
+    if not request.user.is_staff:
+        return JsonResponse({'status': 401,
+                             'msg': 'Unauthorized'})
+
+    inviter = request.user if request.user.is_authenticated else None
+    skills = request.POST.get('skills')
```

Comment:
I think this need to be not only skills, we need to cover all the others filters and check if at least there is one skill selected as a validation.
Because if not, the user will use the other filters think is sending just for some users but really will be sending for more users than that.